main/inc/lib/SessionFieldOption.class.php

```diff
@@ -0,0 +1,8 @@
+<?php
```

Comment:
Name class should be SessionFieldOption.php the same for the rest of files in this PR.